src/main/java/com/google/codeu/servlets/MessageFeedServlet.java

```diff
@@ -33,6 +33,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
     response.setContentType("application/json");
 
     List<Message> messages = datastore.getAllMessages();
+    //try calling calendar here
```

Comment:
Not sure why Github won't let me comment on line 16 above. But is the `@WebServlet("/feed")` working? I don't believe I see the mapping in `web.xml`.

If it is working, that's pretty interesting. Combining both servlet mappings in `web.xml` and via annotations didn't seem possible when I was trying this 😄. I would suggest being consistent and moving that mapping into `web.xml` regardless of whether this is working right now.